### PR TITLE
BUG: Home after markers no longer crash Imviz

### LINKED ISSUE #1964: Imviz: hitting home when markers present returns traceback
**Reporter: Camilla Pacifici**

Load image, zoom, load some markers, do not clear markers, hit home.

[🐱](https://jira.stsci.edu/browse/JDAT-3048)

 --- 

 DISCLAIMER: This issue was autocreated by the Jdaviz Issue Creation Bot on behalf of the reporter. If any information is incorrect, please contact [Duy Nguyen](https://github.com/duytnguyendtn)

### PR DESCRIPTION
**Pull request by pllim**
### Description

This pull request is to prevent crashing in Imviz when Home is clicked on after markers are added.

Fixes #1964 

### Change log entry

- [x] Is a change log needed? If yes, is it added to `CHANGES.rst`? If you want to avoid merge conflicts,
  list the proposed change log here for review and add to `CHANGES.rst` before merge. If no, maintainer
  should add a `no-changelog-entry-needed` label.

### Checklist for package maintainer(s)

This checklist is meant to remind the package maintainer(s) who will review this pull request of some common things to look for. This list is not exhaustive.

- [x] Are two approvals required? Branch protection rule does not check for the second approval. If a second approval is not necessary, please apply the `trivial` label.
- [x] Do the proposed changes actually accomplish desired goals? Also manually run the affected example notebooks, if necessary.
- [x] Do the proposed changes follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [x] Are tests added/updated as required? If so, do they follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [x] Are docs added/updated as required? If so, do they follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [x] Did the CI pass? If not, are the failures related?
- [x] Is a milestone set? Set this to bugfix milestone if this is a bug fix and needs to be released ASAP; otherwise, set this to the next major release milestone.
- [ ] After merge, any internal documentations need updating (e.g., JIRA, Innerspace)?

### CODE REVIEW

File: jdaviz/configs/imviz/tests/test_astrowidgets_api.py [hunk 1]
```diff
@@ -323,6 +323,10 @@ def test_mvp_markers(self):
         # TODO: How to check imviz.app.data_collection.links is correct?
         assert len(self.imviz.app.data_collection.links) == 14
 
+        # Just want to make sure nothing crashes. Zooming already testing elsewhere.
+        # https://github.com/spacetelescope/jdaviz/pull/1971
+        self.viewer.zoom_level = 'fit'
```
Comment (pllim) [inline jdaviz/configs/imviz/tests/test_astrowidgets_api.py:328]: For the record, I did confirm locally that this test fails on `main` without this patch.

**Review by camipacifici — APPROVED** [review]
Works! Thank you!

**Review by kecnry — APPROVED** [review]
Elegant fix, thanks!

**Comment by pllim** [comment]
The dev failure is unrelated. I thought numpy-dev fixed that `__code__` stuff already, but not our problem regardless.

Thanks for the reviews!

**Pull request merged**